### PR TITLE
Microbial Biotechnology Page

### PR DESCRIPTION
**Pull request by davidcmarkham**
Added content on the Microbial Biotechnology page
Relevant tools & resources table needs a tag for this page before it can be included

### CODE REVIEW

File: pages/your_domain/microbial_biotechnology.md [hunk 1]
```diff
@@ -12,27 +12,143 @@ In the event that no adequate Problem page exists for a problem that can be gene
...
 
 ### Solutions
 <!--- Detail, either in normal text or in bullet points, the domain-specific solutions to the problem. Do not merely list tools or resources, as they will be automatically listed in the bottom section, but you can and should mention tools and resources listed below if you detail their usage to solve the problem. --->
 
-<!--- ## Section 2 Title --->
+#### Biological hosts
 <!--- Add more sections as needed, with the same subsections as above. --->
-...
+For microbial biotechnology, bacteria and fungi make up the vast majority of hosts used in the biotechnology industry. These organisms are either sourced from a culture collection, another lab, developed in house or isolated from nature. Hosts will need to be described accordingly to record information about their source and their taxonomy. 
+
```
Comment (smza) [inline pages/your_domain/microbial_biotechnology.md:40]: The structure that needs to be followed is : Description, consideration and solutions.
Could you please add "considerations" section here. For guidance you can have a look at other domain pages

File: pages/your_domain/microbial_biotechnology.md [hunk 2]
```diff
@@ -12,27 +12,143 @@ In the event that no adequate Problem page exists for a problem that can be gene
...
+The genetic properties of a host is typically specified as part of a study where the host is modified or used to harbour a genetic construct. The description of the host can therefore be expanded from the taxonomic information to information about which genes have been inserted into the host genome, which genes have been mutated and the parental information about the strain. A set of genetic nomenclature standards have been established by microbiologists and have been used for many years. These are still a useful way of communicating data about the genotype of a strain, see -  (Maloy and Hughes, 2007).  Databases such as [CellRepo](https://cellrepo.herokuapp.com/) allow strains that have been barcoded to be tracked using a version control type system (Tellechea-Luzardo et al., 2020).
```
Comment (smza) [inline pages/your_domain/microbial_biotechnology.md:47]: This is very detailed. Some of the text is what can be moved to the "consideration" section.

File: pages/your_domain/microbial_biotechnology.md [hunk 3]
```diff
@@ -12,27 +12,143 @@ In the event that no adequate Problem page exists for a problem that can be gene
...
+
+Nowadays the ultimate detailed specification of a host begins with its complete genome sequence. This information should be captured at the nucleotide level using the well-known [European Nucleotide Archive standard (ENA)](https://www.ebi.ac.uk/ena/browser/home) and [submitted](https://www.ebi.ac.uk/ena/browser/submit) to the ENA database to allow the information to be shared. Note that capturing the metadata describing a genome sequence and its host is vitally important to facilitate further studies in comparative genomics and phenotypic analysis. Minimal information standards have been established to specify this metadata, such as the MIGS standard - (Field et al., 2008). The [International Nucleotide Sequence Database Collaboration](http://www.insdc.org/) database collection provides an umbrella for gathering and sharing a variety of sequence data from different sequence databases internationally. Other databases such as Genbank and DDBJ also cater for sequence information.
+
+#### Genetic parts, device and systems
```
Comment (smza) [inline pages/your_domain/microbial_biotechnology.md:51]: This section also needs to be divided into Description, Consideration and Solution.

**Review by smza — CHANGES_REQUESTED** [review]
There is detailed information about each stage of the DBLT cycle. What needs to be done is to dive each stage of the cycle according to the structure of the website for the sake of consistency.

**Comment by davidcmarkham** [comment]
I have tried to break up the text into Description, Considerations & Solutions sections. Can you let me know if this is the sort of thing you had in mind? I can make further alterations where needed but I don't think I can create a single Description, Considerations & Solutions section for the Design section without completely re-writing the the text.

**Pull request merged**